docs/vectorgrid-api-docs.html

```diff
@@ -386,9 +386,9 @@ <h3 id='symbolizer-method'>Methods</h3>
             }
         },
         // A function for styling features dynamically, depending on their
-        // properties, the map&#39;s zoom level, and the layer's geometry type
-        // (point, line, polygon)
-        water: function(properties, zoom, type) {
+        // properties, the map&#39;s zoom level, and the layer's geometry
+        // dimension (point, line, polygon)
+        water: function(properties, zoom, geometry_dimension) {
```

Comment:
I think this should be `geometryDimension`, since we prefer camel case (like most JavaScript code).